platform/src/main/java/org/hillview/storage/CassandraDatabase.java

```diff
@@ -181,67 +236,60 @@ private void setStoredTableInfo() {
         for (KeyspaceMetadata kMetadata : keyspaces) {
             keyspace = kMetadata.getName();
             tables = this.metadata.getKeyspace(keyspace).getTables();
-            tableNames = tables.stream().map(AbstractTableMetadata::getName)
-                .collect(Collectors.toList());
+            tableNames = tables.stream().map(AbstractTableMetadata::getName).collect(Collectors.toList());
             this.cassTables.add(new CassTable(keyspace, tableNames));
         }
     }
 
-    /** Returns list of tables stored in the cluster  */
+    /** Returns list of tables stored in the cluster */
     public List<CassTable> getStoredTableInfo() {
         return this.cassTables;
     }
 
     /**
-     * Given the keyspace and table name, sstableutil will return the sstable-related files.
-     * Using sstableutil command is better than doing manually sstable search. Although we know
-     * the sstable parent dir from cassandra.yaml, we need to rule out the outdated sstables.
-     * And sstableutil is built specifically to identify which version are the most updated.
-      */
-    public String getSSTablePath() {
-        boolean isWindows = System.getProperty("os.name")
-            .toLowerCase().startsWith("windows");
-        Path ssTableUtilPath = Paths.get(ssTableUtilDir);
-        Path cassandraPath = Paths.get(info.cassandraRootDir);
+     * Given the keyspace and table name, sstableutil will return the
+     * sstable-related files. Using sstableutil command is better than doing
+     * manually sstable search. Although we know the sstable parent dir from
+     * cassandra.yaml, we need to rule out the outdated sstables. And sstableutil is
+     * built specifically to identify which version are the most updated.
+     */
+    public List<String> getSSTablePath() {
+        List<String> result = new ArrayList<>();
+        boolean isWindows = System.getProperty("os.name").toLowerCase().startsWith("windows");
```

Comment:
Please move this line into a separate static function in the class Utilities. Call it "runningOnWindows()".